### ISSUE TITLE
Edit Row

### ISSUE THREAD

**Issue by CaverBruce** [richsmith/sexytopo#87]
Tried using ST in a simulated real survey scenario for the first time over the weekend. And took the opportunity to start training a friend in the principles of paperless surveying (using ST v1.2.2).
When choosing Edit Row from the data table, there are two problems.
The first and most significant is that you cannot edit the 'from' or 'to' stations. This is easily accommodated in PocketTopo by clicking the + and - buttons (ie they zoom in graphic views, and increment/decrement the station number in table view).

The second is that the interface re-presents the data in a different format to that in the table (ie what was left to right becomes top to bottom).  A preferred approach would be to edit the table cells in place, without the dialog.

**Comment by richsmith** [comment]
I agree that would be better, but I couldn't find any library to do that at the time I wrote it. May be worth revisiting.

**Comment by CaverBruce** [comment]
Set a friend off as lead surveyor on a multi-day trip the other day to a brand new unsurveyed cave.  Experienced surveyor, but never done it electronically before.  Came back full of enthusiasm for distos and SexyTopo.  On teasing out his experiences, he had two quibbles.
1. A few times he had to delete a few legs worth of data and resurvey the passage, simply because it is not possible to edit the from and to stations for a survey leg.
2. Will mention a another topic.

**Comment by richsmith** [comment]
Good news on the positive feedback!

Editing the to and from stations is of course possible, but there are the specific move and rename functions. I might look into seeing what it would be like to edit them directly (maybe a dropdown for the "from" station and free text for the "to").

**Comment by CaverBruce** [comment]
#172
